doc/btp_gap.txt

```diff
@@ -510,6 +510,115 @@ Commands and responses:
+
+	Opcode 0x23 - Periodic Advertising Start command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	<none>
+		Return Parameters:	<none>
+
+        This command is used to start advertising on Periodic Advertising
+        instance.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x24 - Periodic Advertising Set Advertising Data command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	Advertising data length (2 octets)
+                    [array] Advertising data
+		Return Parameters:	<none>
+
+        This command is used to set advertising data for Periodic Advertising
+        instance. Data is an array of tuples in form of
+        (Ad Type, data length, data),
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x25 - Create Periodic Advertising Sync command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+		                    Address (6 octets)
+					        Advertiser Set ID (1 octet)
+					        Skip (2 octets)
+					        Synchronization Timeout (2 octets)
+					        Flags (1 octet)
+		Return Parameters:	<none>
+
+        This command is used to perform Synchronization procedure with
+        periodic advertiser. Skip determines maximum number of periodic
+        advertising events that controller can skip after a successful
+        reception. Synchronization Timeout is in 10 ms units. Reports Disabled
+        determines if sync creation will lead to IUT sending reports.
+
+		Possible values for the Flags parameter are a bit-wise OR
+		of the following bits:
+					0 = Reports Disabled
+					1 = Duplicate filtering enabled
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x26 - Periodic Advertising Set Transfer Info command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	Address Type (1 octet)
+		                    Address (6 octets)
+					        Service Data (2 octets)
+		Return Parameters:	<none>
+
+        This command is used to initialize set info transfer procedure to peer
+        given by address.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x27 - Periodic Advertising Transfer Start command/response
+
+        Controller Index:	<controller id>
+		Command parameters:	Sync Handle (2 octets)
+		                    Address Type (1 octet)
+		                    Address (6 octets)
+					        Service Data (2 octets)
+		Return Parameters:	<none>
+
+        This command is used to initialize periodic transfer procedure to peer
+        given by address. It is used to transfer periodic sync information
+        to synchronization identified by handle.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x28 - Periodic Advertising Transfer Receive command/response
+
+        Controller Index:	<controller id>
+		Command parameters: Sync Handle (2 octets)
```

Comment:
sync handle is not needed in this command

Reply:
ok, but `Skip` is needed here too